sections/07_conclusions.md

```diff
@@ -91,22 +91,41 @@ very large collections of input data into interpretable models that spur
 scientists to ask questions that they didn't know how to ask, it will be clear
 that deep learning has transformed biology and medicine.
 
+### Continuous collaborative manuscript drafting
```

Comment:
I like that idea.  Should we keep `Author contributions` and `Acknowledgements` in `07_conclusions.md` or move those to a new `09_contributions`?  I think having Methods follow Conclusions directly works well if it is prefaced by a `## Methods` subheading.